h2o-docs/src/product/blog-releases.rst

```diff
@@ -577,3 +577,88 @@ Prior Release Blogs
 ~~~~~~~~~~~~~~~~~~~
 
 You can find all prior release blogs `here <https://h2o.ai/blog/category/h2o-release/>`__.
+
+General Blogs
+-------------
+
+A Look at the UniformRobust method for ``histogram_type``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We introduced the `UniformRobust method for histogram <data-science/algo-params/histogram_type.html>`__ (``histogram_type="UniformRobust"``) in 3.36.1. and want to highlight its benefits! This method addresses issues found in the default UniformAdaptive method and preserve the runtime performance advantages compared to the QuantilesGlobal method.
```

Comment:
I think this uses our own terminology (QuantilesGlobal, UniformRobust) so this message will be missed on non-H2O employees or power users. 

The gist is that we have a new parameter that can increase accuracy for datasets with a lot of outliers without causing significant increase in time.